<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditColorThreshold</class>
 <widget class="QDialog" name="CEditColorThreshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CEditColorThreshold</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,0">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Value</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Var</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Mode</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>FlashingTime</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>FlashingColor</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Conform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
